python/cudf/cudf/io/json.py

```diff
@@ -53,19 +53,39 @@ def _get_cudf_schema_element_from_dtype(
 
 
 def _to_plc_compression(
-    compression: Literal["infer", "gzip", "bz2", "zip", "xz", None],
+    compression: Literal[
+        "brotli",
+        "bz2",
+        "gzip",
+        "infer",
+        "lz4",
+        "lzo",
+        "snappy",
+        "xz",
+        "zib",
```

Comment:
Typo? Also re-sort this (zip before zlib).
```suggestion
        "zlib",
```